labman/db/process.py

```diff
@@ -3293,14 +3336,37 @@ def generate_prep_information(self):
                     result[nk] = val
 
                 # format some final fields
-                result['platform'] = equipment[
-                    result.pop('platform_id')]['description']
+                result['platform'] = 'Illumina'
```

Comment:
This also seems surprising since we do have Nanopores in the lab, and it's of course possible other sequencing platforms will be used in the future

Reply:
I'm surprised in the other direction, since here's the exchange I had when gathering specs for this field:
"Is this always Illumina?"
"Yes we don't have another platform we process although there are others in Qiita"

If we DO need to make some decision by sequencing technology, we will need to store that info in the labman database, and it is not there right now--for example, the "description" field that was previously referenced holds the human-readable NAME of the equipment type being used ("MiSeq", "HiSeq3000", etc) not the sequencing technology.

Reply:
Ok, I've added #342 Support generating prep info files for non-Illumina sequencing runs

Reply:
Thanks